framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2204,12 +2212,64 @@ public ParameterizedExecutableType methodFromUse(
 
     if (tree.getKind() == Tree.Kind.METHOD_INVOCATION
         && TreeUtils.isMethodInvocation(tree, objectGetClass, processingEnv)) {
-      adaptGetClassReturnTypeToReceiver(methodType, receiverType);
+      adaptGetClassReturnTypeToReceiver(methodType, receiverType, tree);
     }
 
     return new ParameterizedExecutableType(methodType, typeargs);
   }
 
+  /**
+   * Apply capture conversion to the type arguments of a method invocation.
+   *
+   * @param typeVarMapping mapping from type variable to its argument
```

Comment:
The `@param` Javadoc merely restate the type, which does not supply any information.  Please try to provide more information, such as what it represents or where it is from or other facts about it.  That will make the code easier to understand.